python/dolfinx/fem/element.py

```diff
@@ -160,3 +162,185 @@ def _(e: basix.finite_element.FiniteElement):
         return CoordinateElement(_cpp.fem.CoordinateElement_float32(e._e))
     except TypeError:
         return CoordinateElement(_cpp.fem.CoordinateElement_float64(e._e))
+
+
+class FiniteElement:
+    _cpp_object: typing.Union[_cpp.fem.FiniteElement_float32, _cpp.fem.FiniteElement_float64]
+
+    def __init__(
+        self,
+        cpp_object: typing.Union[_cpp.fem.FiniteElement_float32, _cpp.fem.FiniteElement_float64],
+    ):
+        """Creates a Python wrapper for the exported finite element class.
+
+        Note:
+            Do not use this constructor directly. Instead use :func:`finite_element`.
```

Comment:
We're not using _ on e.g. functionspace - probably should be consistent.

Reply:
I am not sure what you are referring to, could you elaborate further?

Reply:
The function to create FiniteElement should be called finiteelement to be consistent with e.g. functionspace.

Reply:
I see, thanks.